plugin.video.lacapitv/default.py

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+# KodiAddon
+#
+from resources.lib.scraper import myAddon
+import re
+import sys
+
+# Start of Module
+
+addonName = re.search('plugin\://plugin.video.(.+?)/',str(sys.argv[0])).group(1)
```

Comment:
please drop this, your addon id is a fixed string "plugin.video.lacapitv". Initialize myAddon("plugin.video.lacapitv") instead